### PR TITLE
Vulnerability report template enhancements

### PR DESCRIPTION
**Pull request by mkholjuraev**
Enhances Vulnerability templates:

1. Overlapping the Footer and main table have been properly aligned by reducing the footer font size.
2. Vulneravility advanced report table rows now has 'known exploit' and 'security' labels alongside the ID.
3. CVEs report description is now dynamic, does not show static numbers
4. VulnerabilityExecutiveReportTemplate is not landscape anymore

BEFORE THE ENHANCEMENT:

[new-report (3).pdf](https://github.com/RedHatInsights/pdf-generator/files/11800375/new-report.3.pdf)

AFTER THE ENHANCEMENT: 
[new-report (41).pdf](https://github.com/RedHatInsights/pdf-generator/files/11800349/new-report.41.pdf)

### CODE REVIEW

File: public/templates/footer-template.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     * {
       font-family: 'Red Hat Text';
       font-style: italic;
-      font-size: 9px;
+      font-size: 8px;
```
Comment (Hyperkid123) [inline public/templates/footer-template.html:8]: I know the value might not make sense at first glance, however, these values were taken from UX designs. You can create a custom footer if you require this change.

Comment (mkholjuraev) [reply public/templates/footer-template.html:8]: Aha, but having 8px font will cause the main content to overlap over the footer everywhere. Do we still want to keep 9px as default?

Comment (Hyperkid123) [reply public/templates/footer-template.html:8]: AFAIK It does not overlap in other reports. You might want to change the bottom margin of your template. There is a `browserMargins` config available when registering your template to the template mapper.

Comment (mkholjuraev) [reply public/templates/footer-template.html:8]: Yeah, initially I opted for changing the margin. But it did not give any effect. Anyways, if we see such a problem in the future, we know the solution. I will create a custom footer.

**Comment by mkholjuraev** [comment]
@Hyperkid123 I have updated the test screenshots. Tests are passing locally, but here do not. Need your help.

**Pull request closed without merging**